core/templates/dev/head/pages/admin-page/admin-page.scripts.ts

```diff
@@ -16,7 +16,7 @@
  * @fileoverview Directives required in admin panel.
  */
 
-require('AppInit.ts');
+require('pages/admin-page/admin-page.module.ts');
```

Comment:
Alphabetize this list.

Reply:
As mentioned in chat, please comment why this line must come first.

Reply:
Do this for *all* occurrences of this, I've only noted some of the occurrences.

Reply:
Done!